lib/internal/Magento/Framework/Stdlib/DateTime.php

```diff
@@ -70,7 +70,7 @@ public function formatDate($date, $includeTime = true)
      */
     public function isEmptyDate($date)
     {
-        return preg_replace('#[ 0:-]#', '', $date) === '';
+        return !$date || preg_replace('#[ 0:-]#', '', $date) === '';
```

Comment:
Hello @fascinosum 
Thank you for your proposition
However, I think no, because this way helps us to avoid redundant operation like this `preg_replace('#[ 0:-]#', '', '') === ''`